projects/angular/src/data/stack-view/_stack-view.clarity.scss

```diff
@@ -199,154 +70,77 @@
-      // floats or inline-blocks.
       flex: 1 1 auto;
       display: flex;
+      align-items: center;
 
       // ATM
       // before section is caret placeholder remove it and replace it with pure padding right
-      // padding-right: var(--cds-global-space-9);
+      // padding-right: tokens.$cds-global-space-9;
       &::before {
         display: inline-block;
         content: '';
         float: left;
-        @include mixins.equilateral-var(
-          clr-stack-view-stack-block-caret-dimension,
-          variables.$clr_baselineRem_0_583,
-          variables.$clr-use-custom-properties
-        );
-        @include mixins.css-var(
-          margin,
-          clr-stack-view-block-label-before-margin,
-          (variables.$clr_baselineRem_0_25 + variables.$clr_baselineRem_1px) variables.$clr_baselineRem_0_2 0 0,
-          variables.$clr-use-custom-properties
-        );
+        @include mixins.equilateral(tokens.$cds-global-space-7);
+        margin: tokens.$cds-global-space-3 tokens.$cds-global-space-5 0 0;
         text-align: center;
       }
 
       &:focus {
-        outline: variables.$clr_baselineRem_1px auto -webkit-focus-ring-color;
-        outline-offset: -(variables.$clr_baselineRem_1px);
+        outline-color: Highlight;
+        outline-color: -webkit-focus-ring-color;
+        outline-style: auto;
+        outline-width: tokens.$cds-global-space-1;
+        outline-offset: calc(-1 * tokens.$cds-global-space-1);
       }
     }
 
     .stack-view-key {
       flex: 0 0 40%;
       max-width: 40%;
-
-      @include mixins.css-var(margin, clr-stack-view-key-margin, 0, variables.$clr-use-custom-properties);
+      margin: 0 tokens.$cds-global-space-7 0 0;
     }
 
     .stack-block-caret {
-      @include mixins.equilateral-var(
-        clr-stack-view-stack-block-caret-dimension,
-        variables.$clr_baselineRem_0_583,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        margin-right,
-        clr-stack-view-stack-block-caret-margin-right,
-        variables.$clr_baselineRem_0_2,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        fill,
-        clr-stack-view-stack-block-caret-color,
-        stack-view-variables.$clr-stack-view-stack-block-caret-color,
-        variables.$clr-use-custom-properties
-      );
+      @include mixins.equilateral(tokens.$cds-global-space-7);
+      margin-right: tokens.$cds-global-space-5;
+      fill: stack-view-variables.$clr-stack-view-stack-block-caret-color;
     }
 
     .stack-block-content {
-      @include mixins.css-var(
-        color,
-        clr-stack-view-stack-block-content-text-color,
-        stack-view-variables.$clr-stack-view-stack-block-content-text-color,
-        variables.$clr-use-custom-properties
-      );
       flex: 1 1 auto;
       width: 60%;
       margin-bottom: 0;
-      font-weight: stack-view-variables.$clr-stack-view-font-weight;
```

Comment:
Same as previous. Were we overriding different NG setting with this?
Why did we need it?

Reply:
Docs added in commit comment removed in commit be6430d